docs/rfcs/active/dashboards/1778-dashboards.md

```diff
@@ -0,0 +1,117 @@
+- Start Date: 2020-08-03
+- RFC PR: https://github.com/linkedin/datahub/pull/1778
+- Implementation PR(s): https://github.com/linkedin/datahub/pull/1775
+
+# Dashboards
+
+## Summary
+
+Adding support for dashboards (and charts) metadata cataloging and enabling search & discovery for them.
+The design should accomodate for different dashboarding ([Looker](www.looker.com), [Redash](www.redash.io)) tools used within a company.
+
+## Motivation
+
+Dashboards are a key piece within a data ecosystem of a company. They are used by different groups of employees across different organizations.
+They provide a way to visualize some data assets (tracking datasets or metrics) by allowing slice and dicing of the input data source.
+When a company scales, data assets including dashboards gets richer and bigger. Therefore, it's important to find and access to the right dashboard.
```

Comment:
Maybe you should explicitly have a list of requirements and non requirements? Access is pretty important, I think it is worth explicitly stating what we're trying to do here.

It seems like we're just trying to surface that information so it can be query-able (find all publish dashboards). We are NOT trying to enforce it. Additionally, we are NOT trying to build this so that a preview may eventually be embedded in the frontend, because there may be access restrictions.

tl;dr We don't want to be in the business of enforcing access to dashboards (yet, maybe ever?), which also means no previews.

Reply:
> Maybe you should explicitly have a list of requirements and non requirements?

I can convert to that structure but I believe RFC template should dictate the standard for the RFCs. Actually, template (and questions) helped me a lot while writing this.

> Is that assumption correct?

That's correct. I can add these.

Reply:
> I can convert to that structure but I believe RFC template should dictate the standard for the RFCs. Actually, template (and questions) helped me a lot while writing this.

I agree. I can put together another PR for that.